src/model/managing.officer.corporate.model.ts

```diff
@@ -11,7 +11,9 @@ export const ManagingOfficerCorporateKeys: string[] = [
   "is_on_register_in_country_formed_in",
   "public_register_name",
   "registration_number",
-  "start_date"
+  "start_date",
+  "contact-full-name",
```

Comment:
could you change the hyphens to underscores for the 2 new contact fields? might just be a typo